ISLE/isleapp.cpp

```diff
@@ -658,6 +659,7 @@ MxResult IsleApp::SetupWindow()
 	SDL_SetStringProperty(props, SDL_PROP_WINDOW_CREATE_TITLE_STRING, WINDOW_TITLE);
 #ifdef MINIWIN
 	SDL_SetBooleanProperty(props, SDL_PROP_WINDOW_CREATE_OPENGL_BOOLEAN, true);
```

Comment:
I can't exactly remember why but on 3DS this property would cause SDL to fail to make the window. For platforms where OGL isn't an option, does this automatically get disabled?

On the 3DS port I currently have this line commented out